appinfo/info.xml

```diff
@@ -15,7 +15,7 @@
     <category>monitoring</category>
     <bugs>https://github.com/nextcloud/serverinfo/issues</bugs>
     <dependencies>
-		<nextcloud min-version="18" max-version="18" />
+		<nextcloud min-version="17" max-version="18" />
```

Comment:
Hey :)
For the stable 17 version, please check the `stable17` branch. This should stay 18-18

Reply:
yeah, sorry, didn't see that. This issue has been resolved in a future commit